hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/ContentSummary.java

```diff
@@ -466,7 +466,9 @@ private String formatSize(long size, boolean humanReadable) {
    * @return Constant-width String representation of Erasure Coding Policy
    */
   public String toErasureCodingPolicy() {
-    return String.format(ERASURECODING_POLICY_FORMAT, erasureCodingPolicy);
+    return String.format(ERASURECODING_POLICY_FORMAT,
+        erasureCodingPolicy.equals("Replicated")
```

Comment:
IIUC, `erasureCodingPolicy` is an empty string when it is replicated.
```suggestion
        erasureCodingPolicy.isEmpty()
```

Reply:
Sorry, I might have misunderstood. "Replicated" may be correct.

Reply:
Thanks, i have test replicated file, it shows Replicated.
I reference to the code of Count.java before HDFS-16018

Reply:
Thanks for your explanation.